metadata-utils/src/main/java/com/linkedin/metadata/authorization/PoliciesConfig.java

```diff
@@ -321,6 +321,11 @@ public class PoliciesConfig {
       "Restore Indicies API",
       "The ability to use the Restore Indices API.");
 
+  public static final Privilege GET_TIMESERIES_INDEX_SIZES_PRIVILEGE = Privilege.of(
```

Comment:
This should get added under the default privileges for admin as well in policies.json